### PR TITLE
Initial swag at h7 examples.

### PR DESCRIPTION
**Pull request by bobmcwhirter**

### CODE REVIEW

File: examples/stm32h7/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[package]
+authors = ["Dario Nieuwenhuis <dirbaio@dirbaio.net>"]
+edition = "2018"
+name = "embassy-stm32h4-examples"
```
Comment (Dirbaio) [inline examples/stm32h7/Cargo.toml:4]: stm32h4?

File: examples/stm32h7/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+defmt-error = []
+
+[dependencies]
+embassy = { version = "0.1.0", path = "../../embassy", features = ["defmt", "defmt-trace"] }
+embassy-traits = { version = "0.1.0", path = "../../embassy-traits", features = ["defmt"] }
+embassy-stm32 = { version = "0.1.0", path = "../../embassy-stm32", features = ["defmt", "defmt-trace", "stm32h743zi"]  }
+embassy-extras = {version = "0.1.0", path = "../../embassy-extras" }
+stm32h7 = { version = "0.13", features = ["stm32h743"]}
+stm32h7xx-hal = { version = "0.9.0", features = ["stm32h743"] }
+#stm32l4 = { version = "0.13", features = ["stm32l4x5" ] }
```
Comment (Dirbaio) [inline examples/stm32h7/Cargo.toml:26]: remove dead code

**Review by Dirbaio — COMMENTED** [review]
Cool! Commented a few minor nits.

I think you're missing a `.cargo/config` too

File: examples/stm32h7/Cargo.toml (hunk 1 above)
Comment (bobmcwhirter) [reply examples/stm32h7/Cargo.toml:4]: blah, will fix

File: examples/stm32h7/Cargo.toml (hunk 2 above)
Comment (bobmcwhirter) [reply examples/stm32h7/Cargo.toml:26]: right on.

**Comment by lulf** [comment]
FWIW, h7 should already support clock config using embassy, see src/rcc/h7/mod.rs

**Review by Dirbaio — APPROVED** [review]

**Pull request merged**